src/main/java/uk/gov/companieshouse/api/testdata/config/WebConfig.java

```diff
@@ -9,35 +9,12 @@
 @Configuration
 public class WebConfig implements WebMvcConfigurer {
 
-    private static final String USERS_ENDPOINTS = "/test-data/user/**";
-    private static final String IDENTITY_ENDPOINTS = "/test-data/identity/**";
-    private static final String ACSP_ENDPOINTS = "/test-data/acsp-members/**";
-    private static final String APPEALS_ENDPOINTS = "/test-data/appeals/**";
-    private static final String ACCOUNT_PENALTIES_ENDPOINTS = "/test-data/penalties/**";
-    private static final String USER_COMPANY_ASSOCIATION_ENDPOINTS =
-            "/test-data/associations/**";
-    private static final String TRANSACTIONS_ENDPOINTS = "/test-data/transactions/**";
-    private static final String CERTIFICATES_ENDPOINTS = "/test-data/certificates/**";
-    private static final String CERTIFIED_COPIES_ENDPOINTS = "/test-data/certified-copies/**";
-    private static final String MISSING_IMAGE_DELIVERIES = "/test-data/missing-image-deliveries/**";
-    private static final String SIC_CODE_KEYWORD = "/test-data/combined-sic-activities/**";
-    private static final String ADMIN_PERMISSIONS_ENDPOINTS = "/test-data/admin-permissions/**";
+    private static final String TEST_DATA_INTERNAL_ENDPOINTS = "/test-data/internal/**";
```

Comment:
This looks fine but please double check that they are not accessible - do we have Karate tests to prove that?  If not we should have them